### ISSUE TITLE
Missing Footnote

### ISSUE THREAD

**Issue by rfpng** [sillsdev/ptx2pdf#617]
I created a printout with PTXprint 2.1.1 of Genesis 4-22. In 22:18, which is on the last page of the printout, we have a footnote in Paratext which is being ignored by PTXprint. 
As a test I made a printout of 4-23 and now the missing footnote appears, apparently because now it is no longer in the last page of the printout.
It would be great if this could be fixed!
Thanks,
Roland
[KVGKu-DefaultPTXprintArchive.zip](https://github.com/sillsdev/ptx2pdf/files/8055279/KVGKu-DefaultPTXprintArchive.zip)

**Comment by davidg-sil** [comment]
I can confirm that in some circumstances the footnotes are (still) lost on the last page. A work-around seems to be putting `\pagebreak` at the end of the last book's USFM file.

**Comment by davidg-sil** [comment]
9041eee8  addresses footnotes on unbalanced / unbalancable  pages, and possibly some issues with single column pages. The previously missing footnote in Gen 22:18 is displaying on my computer from 2.1.4 and later. Please can you mark this issue as closed if that's your experience also?

**Comment by rfpng** [comment]
_I replied to the above yesterday by email, but it doesn't seem to have made it into this list._

I finally got around to testing this. The missing footnote is now there, but I was surprised to find the specific verse now no longer being on the last page but on the second to last page.

This is because the **heading at the beginning of chapter 4 is now missing!** See attached PDF file.

Let me know if you need any further details.

Thanks for looking into this

Roland
[ptxprint-Default-GENKVGKu.pdf](https://github.com/sillsdev/ptx2pdf/files/8380025/ptxprint-Default-GENKVGKu.pdf)

**Comment by rfpng** [comment]
Forgot to mention: This was tested with version 2.1.12

**Comment by davidg-sil** [comment]
Using the zip file you gave earlier, I've tried running the code under both 2.1.12  and the current development code, and I'm unable to get the same results as you. I.e. your missing section title is present.

![image](https://user-images.githubusercontent.com/58848232/160901628-d3b5a356-ae35-4cd4-90ab-6a6ec5b4bfbe.png)

* **But** like you, I get the verse on the last-but-one page:

![image](https://user-images.githubusercontent.com/58848232/160902117-fb1f2812-b72e-481c-839f-1871d496d583.png)

I therefore wonder:
*  if some other configuration changes are causing your loss of section heading,  or 
* If the USFM has been changed at all ? (accidental edit by cat / toddler / over-zealous ptxprint-changes.txt ?)
